src/components/WwwFrame/TheBasketBar.vue

```diff
@@ -10,7 +10,7 @@
 	>
 		<!--eslint-enable max-len -->
 		<kv-button
-			:to="basketLink"
```

Comment:
This cannot be removed as it powers the basket link on all UI pages outside of the CC page.

Reply:
@michelleinez @tiarascoleman I did realize that this is shown on our `/cc` pages when in the mobile view. My apologies for thinking it is hidden previously. It would be be nice to get this tweaked so it works for your page and rest of the site.